README.md

```diff
@@ -1,27 +1,40 @@
-# Project Match
-
+![Image alt text](https://i.imgur.com/qD4SvyR.png "Main Page")
+
+## Introduction
+Project Match is a web app created with the MERN stack with typescript integration. It's primary function is to help programmers and designers match with projects they would be interested in. Project Match will then provide team collaboration tools to support new teams with features such as a dedicated platform for mockup reviews/redlines and group chat.
+
+## Contributors
+* [Francesca Sadikin (PM and Designer)](https://github.com/serpient)
+* [luoto](https://github.com/luoto)
+* [ram](https://github.com/ilvcs)
+* [Eric Miller](https://github.com/ericmiller777)
+
+## Setting up the project
+```
+git clone https://github.com/chingu-voyage4/Bears-Team-11.git
+git remote add upstream https://github.com/chingu-voyage4/Bears-Team-11.git
+npm install -g create-react-app
+npm install
+cd bears-team-11/
+npm start
+```
 ## Running the Server
-`npm run start-server` runs the express server on port `8080`.
+- `npm run start-server` runs the express server on port `8080`.
+- Open Postman app or use your browser to reach `http://localhost:8080/`
+- Connect with the API calls above
 
 Currently using [nodemon](https://github.com/remy/nodemon) to watch and restart server for changes. We are also currently not using babel for trancompilation for the servside code.
 
 ## API Calls
 - Index:  `/`
-- Home:  `/home`. ( only if already loged in otherwise redirects to `/`).
-- Login: `/login?email=ram@hello.com&password=123`.
-- Signup: `/signup?email=hello@hello.com&password=123&username=hello&firstName=yourFirstName&lastName=yourLastName`.
-- SignOut: `/signout`.
-
-Currently using [nodemon](https://github.com/remy/nodemon) to watch and restart server for changes. We are also currently not using babel for trancompilation for the servside code.
+- Home:  `/home` ( only if already loged in otherwise redirects to `/`).
+- Login: `/login` (takes email and password params)
+- Signup: `/signup` (takes first name, last name, username, email, and password params)
+- SignOut: `/signout`
```

Comment:
This should work for now, but as this grow, we should think about putting this into a table format like in our gDocs.

Reply:
we have to add like `/api/home`